zkevm-circuits/src/taiko_super_circuit/test.rs

```diff
@@ -0,0 +1,166 @@
+pub use super::*;
+use ethers_signers::{LocalWallet, Signer};
+use halo2_proofs::{dev::MockProver, halo2curves::bn256::Fr};
+use log::error;
+use mock::{TestContext, MOCK_CHAIN_ID};
+use rand::SeedableRng;
+use rand_chacha::ChaCha20Rng;
+use std::collections::HashMap;
+
+use eth_types::{address, bytecode, geth_types::GethData, Word};
+
+#[test]
+fn super_circuit_degree() {
+    let mut cs = ConstraintSystem::<Fr>::default();
+    SuperCircuit::configure(&mut cs);
+    log::info!("super circuit degree: {}", cs.degree());
+    log::info!("super circuit minimum_rows: {}", cs.minimum_rows());
+    assert!(cs.degree() <= 9);
+}
+
+fn test_super_circuit(block: GethData, circuits_params: CircuitsParams) {
+    let (k, circuit, instance, _) = SuperCircuit::<Fr>::build(block, circuits_params).unwrap();
```

Comment:
yes, it can be combined to general test and use feature to tell the target circuit. changed.

Reply:
Some tests are failed because Anchor circuit asks for anchor tx. @johntaiko Please fix them and this will be updated later.

Reply:
I maybe misunderstood the comment. For unittest here,  now both PSE and ourself have unit test for super circuit, but they need different supporting functions like block_1tx, which builds normal tx in PSE but anchor in ours. So I think it's ok to keep them inside each module to avoid kind of more invasive change (as we need to change unittest code in PSE super circuit mod)? Meanwhile, we can also give more specific names to our test related functions.
And for benchmark, as the bench pointed to our circuit already, do we still need to enable PSE by feature or macro?